Assets/HoloToolkit-Examples/Sharing/SharingService/Scripts/RemoteHeadManager.cs

```diff
@@ -26,6 +26,10 @@ public class RemoteHeadInfo
         /// </summary>
         private Dictionary<long, RemoteHeadInfo> remoteHeads = new Dictionary<long, RemoteHeadInfo>();
 
+        protected override void InitializeInternal()
```

Comment:
Perhaps we use these core scripts to demonstrate the auto resolve.
For example: For RemoteHeadManager.cs we must have SharingStage singleton resolved first.

Reply:
I agree with this.

Something like `SharingStage.AssertIsInitialized();` Should be enough.

Reply:
Looking at this I think we want something a little more then that.  If there is no other reason to have RemoteHeadManager initialize on start then ordering, we should be able to move it's initialization to Initialize internal, as well as SharingStage and just have them both call base.Awake().

Reply:
I've run simple tests in the editor for these changes and it runs with each of the scenes in C:\Git\keluecke-holotoolkit-unity\Assets\HoloToolkit-Examples\Sharing\SharingService\Scenes